upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -319,17 +336,69 @@ function verify_fusion_health() {
 	fi
 }
 
+# Get data cataloging status
+function verify_dcs_health () {
+	print info "Verify Data classification service health."
+	# check operators health
+        unhealthy=0
+        oc get project |grep ibm-data-cataloging > /dev/null
+        if [[ $? -ne 0 ]]; then
+                print info "${CHECK_UNKNOW} ibm-data-cataloging project does not exist. It is possible that service is not installed or failed at very early stage."
+                return 0
+        fi
+        notsuccesscount=$(oc get csv -n ibm-data-cataloging|egrep -v 'Succ|NAME'|wc -l)
+	if [ ${notsuccesscount} -ne 0 ]; then
+                print error "${CHECK_FAIL} ${notsuccesscount} operators for DCS are degraded."
+                unhealthy=1
+                print error "${CHECK_UNKNOWN} Here are failed operators. Use \"oc describe csv <csv name> -n ibm-data-cataloging\" to get more details about failure."
```

Comment:
Looks like CHECK_UNKNOWN - is not defined any where, I believe.

Reply:
thanks, took care of it.